AdminUi/src/AdminUi.Infrastructure.Database.Postgres/Migrations/20230929151518_ClientsOverview.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace AdminUi.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class ClientsOverview : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("""
+                CREATE OR REPLACE VIEW "ClientOverviews" AS
```

Comment:
I know it's similar in case of the IdentityOverview, but we should probably rather use `CREATE VIEW` instead of `CREATE OR REPLACE VIEW`. Just to make sure we don't replace anything by accident. Same for SQL Server.

```suggestion
                CREATE VIEW "ClientOverviews" AS
```